config/locales/views/en.yml

```diff
@@ -325,46 +325,59 @@ en:
+        support: 'contact our Support team'
+        dmca: 'review our DMCA Policy and contact our Legal team'
       reportable:
-        intro: 'Please submit a report via this form if you have not submitted this report within the past sixty (60) days, and you:'
-        hack: believe that your account has been hacked, or that someone is trying to hack your account, or
-        tos: believe you have found content uploaded to the Archive which breaches our Terms of Service, or
-        dmca:
-          claim: would like to lodge a DMCA claim - please review our %{dmca_link}
-          policy: DMCA Policy
-      languages_html: <strong>We can answer Abuse reports in %{list}.</strong> Please allow for an additional delay for responses in any language other than English.
+        intro: 'Potential reasons to submit a report to the Policy & Abuse team:'
+        violation: 'You have encountered content on the Archive that violates the %{tos_link}.'
+        content: 'You would like to know whether specific content is allowed on the Archive.'
+        harassment: 'You have experienced or witnessed harassment on the Archive.'
+        hack: 'You believe that your account was hacked.'
+        suspended: Y'ou have been suspended or your work has been removed, and you have not received an email explaining why.'
+      include:
+        intro: 'What to include in your report description:'
+        username: 'the username of the person you are reporting'
+        links: 'links to any of their other content that you would like to report'
+        quote: 'a quote from or summary of the reported content explaining why it violates the %{tos_link}'
+        sources: 'links to any relevant sources or screenshots'
+        languages: 'We can answer Abuse reports in %{list}.'
+        delay: 'There will be an additional delay for responses in any language other than English.'
+        ao3: 'AO3 Terms of Service'
+      do_not_spam: 'Please only report one user at a time, and do not submit multiple reports about the same user. Our volunteer team is small, so it may take several months for us to get to your report.'
       form:
         legend:
-          abuse: Link and Describe Abuse
+          abuse: 'Link and Describe Abuse'
         landmark:
-          send: Send to Abuse Team
+          send: 'Send to Abuse Team'
         submit:
-          active: Submit
+          active: 'Submit'
         name:
-          label: Your name (optional)
+          label: 'Your name or username (optional)'
         email:
-          label: Your email (required)
-          description: We cannot act on reports without a valid email address.
+          label: 'Your email (required)'
+          description: 'We cannot contact you if the email address you provide is invalid.'
         language:
-          label: Select language (required)
+          label: 'Select language (required)'
         summary:
-          label: Brief summary of Terms of Service Violation (required)
```

Comment:
Looks like the casing also causes issues with https://github.com/otwcode/otwarchive/blob/master/features/other_a/abuse_report.feature#L51